src/clr-core/styles/typography/typography.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Experimental/Typography/Getting Started" />
+
+# Typography
+
+<cds-alert status="danger" closable="false" cds-layout="m-b:md">
+  <cds-alert-content>This is a experimental API and will change. Do not use this in production enviroments!</cds-alert-content>
+</cds-alert>
+
+Clarity Core Typography System provides a flexible API to apply typography
+styles explicitly to elements. This gives full control of the styles of
+an element allowing the use of proper sematic HTML elements.
```

Comment:
`This gives full control of the visual styling of an element separate from the HTML semantics` not `This gives full control of the styles of an element allowing the use of proper sematic HTML elements`